src/Disks/VolumeJBOD.h

```diff
@@ -63,8 +63,20 @@ class VolumeJBOD : public IVolume
     bool are_merges_avoided = true;
 
 private:
-    /// Index of last used disk.
-    mutable std::atomic<size_t> last_used = 0;
```

Comment:
Why drop this lock-free behaviour? It looks cool and fast, you can use mutex just for heap.